docs/src/routes/docs/_layout.svelte

```diff
@@ -1,11 +1,17 @@
 <script>
+  import { Button } from 'attractions';
+  import { HomeIcon, ArrowUpIcon } from 'svelte-feather-icons';
   import Header from 'src/containers/docs/header.svelte';
   import MobileNavigation from 'src/containers/docs/mobile-navigation.svelte';
   import DesktopNavigation from 'src/containers/docs/desktop-navigation.svelte';
-  import { HomeIcon } from 'svelte-feather-icons';
 
   export let segment;
 
+  function scrollToTop() {
+    window.scrollTo(0, 0);
+    // TODO: manage focus
```

Comment:
care to elaborate on this?

Reply:
It should reset the focus on the page to the first focusable element, otherwise keyboard users are unable to use this properly